packages/sitecore-jss-dev-tools/src/resolve-scjssconfig.test.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-unused-expressions */
+import { resolveScJssConfig } from './resolve-scjssconfig';
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+describe('resolve-scjssconfig', () => {
+  let consoleSpy: sinon.SinonSpy;
+
+  beforeEach(() => {
+    consoleSpy = sinon.spy(console, 'error');
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  it('should log and reject when config file not found', async () => {
+    const mockConfigPath = './src/test-data/scjssconfig-notexists.json';
+    const resolveInput = {
+      configPath: mockConfigPath,
+      configName: 'sitecore',
+      assert: true,
+    };
+
+    try {
+      await resolveScJssConfig(resolveInput);
+      expect(true).to.be.false; // should be not reachable
```

Comment:
Don't think this assertion is all that necessary. You could do an assertion on the `err` inside the catch if possible (though looks like this would be undefined), but the log assertion there is good enough.